src/RouterConfig.js

```diff
@@ -24,8 +24,8 @@ import ViewFAQ from './Request/FAQ/ViewAllFAQbyGeneral';
 import ViewFAQAdmin from './Request/Admin/ViewAllFAQbyAdmin';
 import CreateFAQ from './Request/Admin/CreateFAQ';
 import EditFAQ from './Request/Admin/EditFAQ';
-import Login from './Register/login';
-import SignUp from './Register/signUp';
+import Login from './Register/Login';
+import SignUp from './Register/SignUp';
```

Comment:
line 27 and 28, your file names are `login` and `signUp`, so when you import the `./Register/Login`, it can't find the module.